### PR TITLE
Pass next and previous endpoints for a given content topic to iterator

### PR DESCRIPTION
**Pull request by lawong-d2l**
https://github.com/BrightspaceHypermediaComponents/sequences/pull/86

### CODE REVIEW

File: components/sequence-viewer-header.js [hunk 1]
```diff
@@ -170,11 +170,11 @@ class D2LSequenceViewerHeader extends mixinBehaviors([D2L.PolymerBehaviors.Siren
...
 				<d2l-sequences-topic-name id="topicName" class="topic-name col8 hidden-small" href="[[href]]" token="[[token]]" role="heading"></d2l-sequences-topic-name>
 				<div class="col9"></div>
-				<d2l-sequences-iterator class="iterator-icon prev-button col10" href="{{href}}" token="[[token]]" icon="d2l-tier3:chevron-left-circle" previous=""></d2l-sequences-iterator>
+				<d2l-sequences-iterator class="iterator-icon prev-button col10" disabled="[[!previousActivityHref]]" current-activity="{{href}}" href="[[previousActivityHref]]" token="[[token]]" icon="d2l-tier3:chevron-left-circle" previous=""></d2l-sequences-iterator>
 				<div class="col11"></div>
 				<d2l-icon class="flyout-divider col12" icon="d2l-tier2:divider-big"></d2l-icon>
 				<div class="col13"></div>
-				<d2l-sequences-iterator class="iterator-icon next-button col14" href="{{href}}" token="[[token]]" icon="d2l-tier3:chevron-right-circle" next=""></d2l-sequences-iterator>
+				<d2l-sequences-iterator class="iterator-icon next-button col14" disabled="[[!nextActivityHref]]" current-activity="{{href}}" href="[[nextActivityHref]]" token="[[token]]" icon="d2l-tier3:chevron-right-circle" next=""></d2l-sequences-iterator>
```
Comment (jwong-d2l) [inline components/sequence-viewer-header.js]: you could make the iterator smart enough to disable itself if !nextActivityHref, then you wouldn't need to pass the same param twice

Comment (lawong-d2l) [reply components/sequence-viewer-header.js]: Great idea. Resolved

File: package.json [hunk 2]
```diff
@@ -100,7 +100,6 @@
     "d2l-typography": "BrightspaceUI/typography#semver:^7",
     "@polymer/iron-a11y-announcer": "^3.0.0",
     "@polymer/polymer": "^3.0.0",
-    "siren-entity": "BrightspaceHypermediaComponents/siren-entity#semver:^1",
```
Comment (jwong-d2l) [inline package.json]: don't we still need this?

Comment (lawong-d2l) [reply package.json]: @jwong-d2l i didn't find any reference to it

**Pull request merged**